examples/cd_context_20newsgroup.ipynb

```diff
@@ -0,0 +1,3186 @@
+{
```

Comment:
<p>I think the below stems from unfamiliarity with using 'prevalence'</p><p> </p><p>' to a change in relative prevalence of each of these subpopulations' -&gt; ' to a change in the relative prevalences of these subpopulations'</p><p>'A&nbsp;<strong>relative change in prevalence of subpopulations</strong>&nbsp;(i.e. news topics) which are also present in the reference data is observed.' -&gt; </p><p>	'A<strong> change in the prevalences of subpopulations</strong>&nbsp;(i.e. news topics) relative to their prevalences in the training data.' </p><p> </p><p> </p><p>Maybe also:</p><p>'the distribution of each individual subpopulation' -&gt; 'the distribution underlying each subpopulation'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>